testData/solutions/win32.json5

```diff
@@ -4911,6 +4911,300 @@
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "KeyboardPreferenceOn": {
+                        "transform": {
+                            "type": "fluid.transforms.value",
+                            "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.underlineMenuShortcuts.KeyboardPrefEnabled",
+                            "outputPath": "value"
+                        },
+                        "path": {
+                            "transform": {
+                                "type": "fluid.transforms.literalValue",
+                                "input": {
+                                    "get": "pvParam",
+                                    "set": "uiParam"
+                                }
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.underlineMenuShortcuts.KeyboardPrefEnabled": "KeyboardPreferenceOn.value"
+                }
+            }
+        },
+        "configure": [
+            "settings.configureShortcuts",
+            "settings.keyboardPreferred"
+        ],
+        "restore": [
+            "settings.keyboardPreferred",
+            "settings.configureShortcuts"
+        ],
+        "isInstalled": [
+            {
+                "type": "gpii.deviceReporter.alwaysInstalled"
+            }
+        ]
+    },
+
+    "com.microsoft.windows.shortcutWarningMessage": {
+        "name": "Warning messages for shortcuts",
+        "contexts": {
+            "OS": [
+                {
+                    "id": "win32",
+                    "version": ">=5.0"
+                }
+            ]
+        },
+        "settingsHandlers": {
+            "configure": {
+                "type": "gpii.windows.registrySettingsHandler",
+                "liveness": "liveRestart",
+                "options": {
+                    "hKey": "HKEY_CURRENT_USER",
+                    "path": "Control Panel\\Accessibility",
+                    "dataTypes": {
+                        "Warning Sounds": "REG_DWORD"
+                    }
+                },
+                "supportedSettings": {
+                    "Warning Sounds": {
+                        "schema": {
+                            "title": "Shortcut warning messages",
+                            "description": "Enable/Disable warning messages when changing a setting with a shortcut",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "Warning Sounds": {
+                        "transform": {
+                            "type": "fluid.transforms.condition",
```

Comment:
Easier than condition for all of these 0/1 is the valueMapper https://docs.fluidproject.org/infusion/development/ModelTransformationAPI.html#mapping-based-on-input-value-fluidtransformsvaluemapper - and then you won't need to write any inverses

Reply:
Nice, will change it. The same approach is done multiple times in the SR, should I open a ticket to change all that transforms in favor of "fluid.transform.valueMapper"?